### PR TITLE
Update eclipse.json

### PR DESCRIPTION
**Pull request by namandistro**

### CODE REVIEW

**Comment by namandistro** [comment]
@moollaza i am closing other one have a look here.. if u need something let me know

File: share/goodie/cheat_sheets/json/eclipse.json [hunk 1]
```diff
@@ -1,552 +1,464 @@
 {
-    "id": "eclipse_cheat_sheet",
-    "name": "Eclipse IDE",
-    "metadata": {
-        "sourceName": "Shortcut World",
-        "sourceUrl": "http://www.shortcutworld.com/en/win/Eclipse.html"
-        },
-    "section_order": [
+    "id":"eclipse_cheat_sheet",
+    "name":"Eclipse",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/eclipse.json]: Let's keep the name as `Eclipse IDE`

File: share/goodie/cheat_sheets/json/eclipse.json [hunk 2]
```diff
@@ -1,552 +1,464 @@
...
+    "description":"Eclipse Keyboard Shortcuts ",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/eclipse.json]: nit: no need for title casing here: "Basic keyboard shortcuts for the Eclipse IDE"

File: share/goodie/cheat_sheets/json/eclipse.json [hunk 3]
```diff
@@ -1,552 +1,464 @@
...
             {
-                "key": "[Ctrl] [Page Down]",
-                "val": "Switch to next editor"
+                "key":"F12",
+                "val":"Jump to Editor Window"
             },
             {
-                "key": "[Ctrl] [Page Up]",
-                "val": "Switch to previous editor"
+                "key":"[Ctrl] ([↑]/[↓])",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/eclipse.json]: Please make sure these have spaces between them: `( [↑] / [↓] )`

File: share/goodie/cheat_sheets/json/eclipse.json [hunk 4]
```diff
@@ -1,552 +1,464 @@
...
-        "Navigate in Editor": [
-            {
-                "key": "Home",
-                "val": "Jump to beginning of indention. Press home twice to jump to beginning of line"
-            },
+        "Navigate in Editor":[
             {
-                "key": "End",
-                "val": "Jump to end of indention."
+                "key":"([Home]/[End])",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/eclipse.json]: We should keep these ones with alternatives separated for clarity. I.e. All the previous/next, let's keep that as they are, separated

File: share/goodie/cheat_sheets/json/eclipse.json [hunk 5]
```diff
@@ -1,552 +1,464 @@
...
-                "key": "[Ctrl] [Shift] [P]",
-                "val": "With a bracket selected: jump to the matching closing or opening bracket"
-            },
+                "key":"[Alt] ([Page Up]/[Page Down])",
+                "val":"Next Sub-Tab / Previous Sub-Tab"
+            }
+        ],
+        "Select Text":[
             {
-                "key": "[Ctrl] [+]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/eclipse.json]: Looks like this and several other shortcuts are being removed -- are the no longer around in Eclipse? or have they changed? Would just like to know why they're being removed.

**Comment by moollaza** [comment]
@namandistro thanks for updating, I've left some comments and feedback. Please make the necessary fixes and let us know when this is ready for review.

Also, please explain the major differences/discrepancies. It seems like this is removing several shortcuts?

**Comment by moollaza** [comment]
@namandistro looks like this one has conflicts now, please fix and we'll be able to review!

**Pull request closed without merging**